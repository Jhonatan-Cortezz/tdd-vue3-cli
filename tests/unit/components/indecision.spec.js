import { shallowMount } from '@vue/test-utils'
import Indecition from '@/components/Indecition.vue'

describe('Indesicion component', () => {

  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Indecition)
  })

  test('debe de ser match con el snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('escribir en el input no debe de disparar un evento', () => {

  })

  test('escribir "?" debe de disparar el fetch a la api', () => {

  })

  test('pruebas en get answer', () => {

  })

  test('pruebas en getAnswer - fallo api', () => {
    
  })

})