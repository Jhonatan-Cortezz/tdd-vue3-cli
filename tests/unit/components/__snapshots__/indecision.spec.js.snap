// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Indesicion component debe de ser match con el snapshot 1`] = `
<img src="https://via.placeholder.com/250" alt="bg">
<div class="bg-dark"></div>
<div class="indecision-container"><input type="text" placeholder="Hazme una pregunta">
  <p>Recuerda terminar con un signo de interrogaci√≥n</p>
  <div>
    <h2></h2>
    <h1></h1>
  </div>
</div>
`;
